api/RTCStatsReport.json

```
@@ -255,9 +258,8606 @@
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "≤13.1"
+              },
+              "safari_ios": {
+                "version_added": "≤16.4"
+              },
+              "samsunginternet_android": {
+                "version_added": "≤20.0"
+              },
+              "webview_android": {
+                "version_added": "≤112"
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        }
+      },
+      "type_codec": {
+        "__compat": {
+          "description": "<code>codec</code> stats",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCStatsReport#codec",
+          "spec_url": "https://www.w3.org/TR/webrtc-stats/#dom-rtcstatstype-codec",
+          "support": {
+            "chrome": {
+              "version_added": "≤80"
+            },
+            "chrome_android": {
+              "version_added": "≤112"
+            },
+            "edge": {
+              "version_added": "≤80"
+            },
+            "firefox": {
+              "version_added": "98"
+            },
+            "firefox_android": {
+              "version_added": "≤112"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": "≤13.1"
+            },
+            "safari_ios": {
+              "version_added": "≤16.4"
+            },
+            "samsunginternet_android": {
+              "version_added": "≤20.0"
+            },
+            "webview_android": {
+              "version_added": "≤112"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "channels": {
+          "__compat": {
+            "description": "<code>channels</code> in 'codec' stats",
+            "spec_url": "https://w3c.github.io/webrtc-stats/#dom-rtccodecstats-channels",
+            "support": {
+              "chrome": {
+                "version_added": "83"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "98"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "13.1> ≤14.1"
```

https://mdn-bcd-collector.gooborg.com/tests/api/RTCStatsReport/type_codec/channels
- Present 14.1
- Not in 12.1, 13.1,   

First testable build?
```suggestion
                "version_added": "14.1"
```

---

As discussed offline, this is at most off by .1 (i.e. it could be in 14). It is not verifiable right now. Agreed to accept this.